src/mainloop.cpp

```diff
@@ -154,24 +154,48 @@ void Mainloop::route_msg(struct buffer *buf, int target_sysid, int target_compid
     bool unknown = true;
 
     for (Endpoint **e = g_endpoints; *e != nullptr; e++) {
-        if ((*e)->accept_msg(target_sysid, target_compid, sender_sysid, sender_compid, msg_id)) {
+        switch (
+            (*e)->accept_msg(target_sysid, target_compid, sender_sysid, sender_compid, msg_id)) {
```

Comment:
```
AcceptState state = (*e)->accept_msg(target_sysid, target_compid, sender_sysid, sender_compid, msg_id);
switch (state) {
```

would probably be a little more readable and avoid funny autoformat

Reply:
Still results in a somewhat funny autoformat, but might be easier during debugging when the variable has a name.